### DISCUSSION TITLE
Algorithms used in MMPose

### DISCUSSION THREAD

**Discussion by leonweli111** [open-mmlab/mmpose#1922] [Q&A]
Hoping this is the right place for this question but I was wondering for my uni project what are the default algorithms used for MMPose?

**Comment by ly015** [comment]
Hi, thanks for using MMPose. MMPose supports over 30 algorithms for various tasks. You can check out the [demos](https://mmpose.readthedocs.io/en/latest/demo.html)  related to your project and the model used in these examples can be found in the example command line. For example, in the [demo for top-down 2D body keypoint detection](https://mmpose.readthedocs.io/en/latest/demo.html#using-mmdet-for-human-bounding-box-detection), we use a FasterRCNN detector from MMDetection, and an HRNet keypoint model from MMPose.
